apps/admin/src/pages/hackathons/hackathonsTable.tsx

```diff
@@ -0,0 +1,138 @@
+import type { ColumnDef, Row } from "@tanstack/react-table";
+import type { User } from "lucide-react";
+import { useState } from "react";
+import { ArrowUpDown, MoreHorizontal } from "lucide-react";
+import { toast } from "sonner";
+
+import type { RouterOutput } from "@knighthacks/api";
+
+import { Button } from "~/components/ui/button";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from "~/components/ui/dropdown-menu";
+import { Sheet, SheetContent } from "~/components/ui/sheet";
+import { trpc } from "~/trpc";
+import { UpdateHackathonForm } from "./update-hackathon-form";
+
+type Hackathon = RouterOutput["hackathons"]["getAll"][number];
+
+export const columns: ColumnDef<Hackathon>[] = [
```

Comment:
You should remove this and use the columns array exported from `hackathons-columns`.